salalib/shapemap.cpp

```diff
@@ -3815,26 +3837,26 @@ bool ShapeMap::ozlemSpecial4(ValuePair& cut, IntPair& previous, int& state, Attr
 // expects to be the rays layer... check intersection of each ray with *all* O.S. polys
 void ShapeMap::ozlemSpecial5(ShapeMap& buildings)
 {
-   pmap<int,int> exposureranks;
-   exposureranks.add(10172,10);
-   exposureranks.add(10183,9);
-   exposureranks.add(10056,8);
-   exposureranks.add(10054,7);
-   exposureranks.add(10123,6);
-   exposureranks.add(10119,5);
-   exposureranks.add(10111,4);
-   exposureranks.add(10217,3);
-   exposureranks.add(10167,2);
-   exposureranks.add(10089,1);
-   exposureranks.add(10053,0);
-   exposureranks.add(-1,-1);
+   std::map<int,int> exposureranks;

```

Comment:
(Just in case this method can't go wholsesale:) This should be done in an initialiser list.